commons/src/main/java/com/powsybl/commons/compress/SafeZipInputStream.java

```diff
@@ -15,8 +15,8 @@
 
 public class SafeZipInputStream extends ForwardingInputStream<ZipInputStream> {
 
-    private int bytesRead;
-    private int maxBytesToRead;
+    private long bytesRead;
+    private long maxBytesToRead;
 
     public SafeZipInputStream(ZipInputStream in, int entryNumber, int maxBytesToRead) throws IOException {
```

Comment:
You forgot to change maxBytesToRead to a long here. 

Besides it's not clear what entryNumber is here. It's not exactly the number of entries to skip, as when you just created the zipInputStream and give entryNumber = 1 you're not skipping anything. And it's not really something to do with the "safe" responsability of the SafeZipInputStream class. Therefore, could you remove this argument and let the caller do the call(s) to getNextEntry?